runtime/cudaq/platform/default/rest/helpers/braket/BraketExecutor.h

```diff
@@ -0,0 +1,109 @@
+/*******************************************************************************
+ * Copyright (c) 2022 - 2024 NVIDIA Corporation & Affiliates.                  *
+ * All rights reserved.                                                        *
+ *                                                                             *
+ * This source code and the accompanying materials are made available under    *
+ * the terms of the Apache License 2.0 which accompanies this distribution.    *
+ ******************************************************************************/
+
+#pragma once
+#include "common/Executor.h"
+#include "common/FmtCore.h"
+#include "common/MeasureCounts.h"
+#include "cudaq.h"
+
```

Comment:
nit: remove whitespace between these includes.

Reply:
Addressed in PR #2448